HackingWithSwift/week1/day_5/Accepting parameters/Accepting parameters.playground/Contents.swift

```diff
@@ -0,0 +1,13 @@
+func example (number: Int) {
```

Comment:
There should be no space between the function name and the opening parenthesis in Swift.
The correct syntax would be:
```
func example(number: Int) {
    // Function body
}
```